doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -552,90 +517,78 @@ the conversion in hypotheses :n:`{+ @ident}`.
    Example: The term :g:`fun n : nat => S n + S n` is not reduced by :n:`hnf`.
 
 .. note::
- The :math:`\delta` rule only applies to transparent constants (see :ref:`vernac-controlling-the-reduction-strategies`
- on transparency and opacity).
+ The :math:`\delta` rule only applies to transparent constants
+ (see :ref:`controlling-the-reduction-strategies` on transparency and opacity).
+
+.. tacn:: simpl {? @delta_reductions } {? {| @reference_occs | @pattern_occs } } {? @occurrences }
 
-.. tacn:: cbn
-          simpl
-   :name: cbn; simpl
+   .. insertprodn reference_occs pattern_occs
 
-   These tactics apply to any goal. They try to reduce a term to
-   something still readable instead of fully normalizing it. They perform
+   .. prodn::
+      reference_occs ::= @reference {? at @occs_nums }
+      pattern_occs ::= @one_term {? at @occs_nums }
+
+   Reduces a term to
+   something still readable instead of fully normalizing it. It performs
    a sort of strong normalization with two key differences:
 
-   + They unfold a constant if and only if it leads to a :math:`\iota`-reduction,
+   + It unfolds constants only if they lead to an :math:`\iota`-reduction,
      i.e. reducing a match or unfolding a fixpoint.
-   + While reducing a constant unfolding to (co)fixpoints, the tactics
-     use the name of the constant the (co)fixpoint comes from instead of
+   + When reducing a constant unfolding to (co)fixpoints, the tactic
+     uses the name of the constant the (co)fixpoint comes from instead of
      the (co)fixpoint definition in recursive calls.
 
-   The :tacn:`cbn` tactic was intended to be a more principled, faster and more
-   predictable replacement for :tacn:`simpl`.
-
-   The :tacn:`cbn` tactic accepts the same flags as :tacn:`cbv` and
-   :tacn:`lazy`. The behavior of both :tacn:`simpl` and :tacn:`cbn`
-   can be tuned using the :cmd:`Arguments` command.
+   :n:`@occurrences`
+     Permits selecting whether to reduce the conclusion and/or one or more
+     hypotheses.  The `at` option allows selection of specific occurrences
+     matching :n:`@pattern_occs`.
```

Comment:
I would rather say that the `at` clause is disallowed here and may be used through `@reference_occs` / `@pattern_occs`.

Reply:
Homer nods.  Got it.